tests/harness.js

```diff
@@ -52,6 +52,38 @@ fluid.defaults("gpii.tests.testem.harness", {
             priority: "before:cleanup",
             funcName: "gpii.tests.testem.harness.outputOptions",
             args:     ["{that}"]
+        },
+        constructFixtures: {
```

Comment:
Let's namespace all of this set - e.g. with names such as "constructFixtures.log", etc.

Reply:
Done.